config/jobs-chromeos.yaml

```
@@ -435,6 +435,14 @@ jobs:
       branch:
         - for-kernelci
 
+  kselftest-cpufreq:
+    template: kselftest.jinja2
+    kind: test
+    params:
+      nfsroot: 'http://storage.kernelci.org/images/rootfs/debian/bookworm-kselftest/20240313.0/{debarch}'
+      collections: cpufreq
+      job_timeout: 10
+
```

Could you please add `kcidb_test_suite` here? It is a mapping of KCI test with KCIDB test suite. We would require it for submitting results to KCIDB.
```suggestion
  kselftest-cpufreq:
    template: kselftest.jinja2
    kind: test
    params:
      nfsroot: 'http://storage.kernelci.org/images/rootfs/debian/bookworm-kselftest/20240313.0/{debarch}'
      collections: cpufreq
      job_timeout: 10
    kcidb_test_suite: kselftest.cpufreq
```

---

Sure, I'll make this change in v2